src/store/normalize-data.test.js

```diff
@@ -1,4 +1,6 @@
-import normalizeData, { createInitialPipelineState } from './normalize-data';
+import normalizeData, {
+  createInitialPipelineState,
+} from './normalize-data';
```

Comment:
Why has this been changed? That's odd 🤔

Reply:
yea it's a leftover styling change from importing another function ( which was later deleted ) - will revert this back to the old style:)